kafka/cfg.go

```diff
@@ -0,0 +1,64 @@
+package kafka
+
+import (
+	"flag"
+
+	"github.com/Shopify/sarama"
+	"github.com/Shopify/sarama/tools/tls"
+	log "github.com/sirupsen/logrus"
+)
+
+type KafkaNet struct {
```

Comment:
what does the 'Net' mean in the name here and in ConfigNet?
I think this type can be just called 'Cfg' and ConfigNet can probably be NewCfg

Reply:
This only configures the `config.Net.*` parameters, hence the name.